src/styles/variables.scss

```diff
@@ -0,0 +1,13 @@
+:root {
+  /** chip **/
+  --chip-min-width: 7rem;
```

Comment:
i believe we might drop some of variables if we gonna use native tailwind styles or at least we can extend tailwind configuration  with missing values